guides/source/active_record_basics.md

```diff
@@ -71,88 +67,105 @@ Convention over Configuration in Active Record
 
 | Model / Class    | Table / Schema |
 | ---------------- | -------------- |
 | `Article`        | `articles`     |
 | `LineItem`       | `line_items`   |
-| `Deer`           | `deers`        |
+| `Product`        | `products`     |
 | `Mouse`          | `mice`         |
 | `Person`         | `people`       |
 
 ### Schema Conventions
 
-Active Record uses naming conventions for the columns in database tables,
+Active Record uses conventions for column names in the database tables as well,
 depending on the purpose of these columns.
 
+* **Primary keys** - By default, Active Record will use an integer column named
+  `id` as the table's primary key (`bigint` for PostgreSQL and MySQL, `integer`
+  for SQLite). When using [Active Record
+  Migrations](active_record_migrations.html) to create your tables, this column
+  will be automatically created.
 * **Foreign keys** - These fields should be named following the pattern
   `singularized_table_name_id` (e.g., `item_id`, `order_id`). These are the
   fields that Active Record will look for when you create associations between
   your models.
-* **Primary keys** - By default, Active Record will use an integer column named
-  `id` as the table's primary key (`bigint` for PostgreSQL and MySQL, `integer`
-  for SQLite). When using [Active Record Migrations](active_record_migrations.html)
-  to create your tables, this column will be automatically created.
 
-There are also some optional column names that will add additional features
-to Active Record instances:
+There are also some optional column names that will add additional features to
+Active Record instances:
 
 * `created_at` - Automatically gets set to the current date and time when the
   record is first created.
 * `updated_at` - Automatically gets set to the current date and time whenever
   the record is created or updated.
 * `lock_version` - Adds [optimistic
-  locking](https://api.rubyonrails.org/classes/ActiveRecord/Locking.html) to
-  a model.
+  locking](https://api.rubyonrails.org/classes/ActiveRecord/Locking.html) to a
+  model.
 * `type` - Specifies that the model uses [Single Table
   Inheritance](https://api.rubyonrails.org/classes/ActiveRecord/Base.html#class-ActiveRecord::Base-label-Single+table+inheritance).
-* `(association_name)_type` - Stores the type for
-  [polymorphic associations](association_basics.html#polymorphic-associations).
+* `(association_name)_type` - Stores the type for [polymorphic
+  associations](association_basics.html#polymorphic-associations).
 * `(table_name)_count` - Used to cache the number of belonging objects on
-  associations. For example, a `comments_count` column in an `Article` class that
-  has many instances of `Comment` will cache the number of existent comments
-  for each article.
+  associations. For example, a `comments_count` column in an `Article` class
+  that has many instances of `Comment` will cache the number of existent
+  comments for each article.
```

Comment:
Should this be existing instead of existent?

```suggestion
  that has many instances of `Comment` will cache the number of existing
  comments for each article.
```